manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -30,19 +32,19 @@ const getPolkadotSignerAndAddress = async () => {
     }
 }
 
-/// Test to publicly transfer 10 DOL.
+/// Test to publicly transfer 10 KMA.
 const publicTransferTest = async () => {
     const privateWalletConfig = {
         environment: Environment.Development,
-        network: Network.Dolphin
+        network: Network.Calamari
```

Comment:
My reasoning here is that Dolphin.Production is different than Dolphin.Development.  The former is actually Calamari (12 decimals) and the latter is Dolphin (18 decimals)
So using Dolphin in these examples, especially with Development might get confusing. So I changed to Calamari in order to kind of hide this complexity (and potential confusion)
I think if the example is Calamari.Development. like it is now, they should have the KMA, since it's a dev env.